CWE/localisation/Chrisfox's events.csv

```diff
@@ -548,3 +548,8 @@ EVTDESC8120711;Tatarstan continues to work under the leadership of the Supreme S
 EVTDESC8120712;Negotiations with Russia have once again begun on the ultimate fate of Tatarstan. Currently a final ultimatum has been presented � with assurances of self-governance and cultural autonomy being offered in exchange for joining the federation. With the current situation of Tatarstan, and the geography of the region making full independence problematic, many feel it would be in our best interest to accept.;;x;;;;;;;;;;;;;;;;
 EVTDESC8120713;Tatarstan has announced that they are finally ready to sign the Federation Treaty and join the Russian Federation. While they had previously left their ultimate fate up in the air with vague declarations of �sovereignty�, assurances of autonomy have since swayed them. With any hope, this move will convince other republics that they will continue to have a place as equals within Russia.;;x;;;;;;;;;;;;;;;;
 EVTDESC8120714;Even with significant concessions, the Tatarstani leadership have continued to refuse to sign any agreement on a federation with Russia � effectively declaring themselves independent in the process. Such an affront to Russian national unity � now more important than ever as Russia faces political crisis and economic unrest � must be met with force. While Russia�s new army is still untested, the leadership feel that the restoration of order in the region is more than enough to prove its continued might.;;x;;;;;;;;;;;;;;;;
+MRH;Maharashtra;;x;;;;;;;;;;;;;;;;
+MRH_ADJ;Marathi;;x;;;;;;;;;;;;;;;;
+MRH_absolute_monarchy;Marathi Empire;;x;;;;;;;;;;;;;;;;
+MRH_hms_government;Marathi Empire;;x;;;;;;;;;;;;;;;;
```

Comment:
Constitutional monarchies shouldn’t have Empire in their names, following the V2 convention.